packages/wonder-blocks-testing/src/fixtures/fixtures.defaultwrapper.stories.js

```diff
@@ -0,0 +1,59 @@
+// @flow
+import * as React from "react";
+
+import {setupFixtures, fixtures, adapters} from "../index.js";
+
+// Normally would call setup from the storybook.main.js for a project.
+setupFixtures({
+    adapter: adapters.storybook(),
+});
+
+const MyComponent = (props) => `My props: ${JSON.stringify(props, null, 2)}`;
+
+const Wrapper = (props) => (
+    <>
+        Wrapper &gt;&gt;&gt;
+        <MyComponent {...props} />
+        &lt;&lt;&lt; Wrapper
+    </>
+);
+
+const DefaultWrapper = (props) => (
+    <>
+        DefaultWrapper &gt;&gt;&gt;
+        <MyComponent {...props} />
+        &lt;&lt;&lt; DefaultWrapper
+    </>
+);
+
+const stories: Array<mixed> = Object.values(
+    fixtures(
+        {
+            component: MyComponent,
+            title: "Testing/Fixtures/DefaultWrapper",
+            defaultWrapper: DefaultWrapper,
+        },
+        (fixture) => {
+            fixture(
+                "This is a fixture with some regular props and the default wrapper",
+                {
+                    see: "this is a prop",
+                    and: "this is another",
+                },
+            );
+
+            fixture(
+                "This fixture uses a custom wrapper",
+                {
+                    just: "some props again",
+                    like: "this one",
+                },
+                Wrapper,
+            );
+        },
+    ) ?? {},
+);
+
+export default stories[0];
+export const f1 = stories[1];
+export const f2 = stories[2];
```

Comment:
Disappointed that `module.exports = fixtures(...)` was insufficient in WB, but this works to demonstrate things for now. Really wish there was a way to export multiple stories without naming them all upfront.

Reply:
We could create a babel plugin to help with this.  No action req'd in this PR.

Reply:
Yeah - it would need to at least be able to know how many fixtures there are though.